drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -38,6 +38,35 @@ static struct net_mgmt_event_callback ip_maddr4_cb;
 static struct net_mgmt_event_callback ip_maddr6_cb;
 #endif /* CONFIG_NRF700X_STA_MODE */
 
+void nrf_wifi_set_iface_event_handler(void *os_vif_ctx,
+						struct nrf_wifi_umac_event_set_interface *event,
+						unsigned int event_len)
+{
+	struct nrf_wifi_vif_ctx_zep *vif_ctx_zep = NULL;
+
+	if (!os_vif_ctx) {
+		LOG_ERR("%s: Invalid parameters",
+			__func__);
+		goto out;
+	}
+
+	if (!event) {
+		LOG_ERR("%s: event is NULL",
+			__func__);
+		goto out;
+	}
+
+	(void)event_len;
```

Comment:
What does this do?

Reply:
Tells the compiler that `not using this variable is intended and not a bug` and then we don't see `unused variable` warning.